src/ExpEvaluator.h

```diff
@@ -10,9 +10,9 @@
 
 #ifdef __cplusplus
 #define _USE_MATH_DEFINES
-#include <math.h>
 #include "log.h"
 #include "PolarQuad.h"
+#include <math.h>
```

Comment:
Yes it actually is a meaningful change. Including any of the standard C libraries prior to including "Python.h" leads to warning messages at compile time (see my discussion in PR #143 for more details). In this case, "log.h" includes "Python.h" first, which is then included here first. Unfortunately, Git didn't do a good job of ordering the includes when it merged together some of the old PRs.